Packs/DelineaSS/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Thycotic
+- Updated the Docker image to: *demisto/python3:3.9.7.24076*.

```

Comment:
Please remove all Release notes as this is a new pack (while the original will be deprecated.)